hack/install.sh

```diff
@@ -467,6 +467,12 @@ do_install() {
 				( set -x; $sh_c 'sleep 3; apt-get install -y -q gnupg2 || apt-get install -y -q gnupg' )
 			fi
 
+			# dirmngr is a separate package in ubuntu yakkety; see https://bugs.launchpad.net/ubuntu/+source/apt/+bug/1634464
+			if [ ! -e /usr/bin/dirmngr ]; then
```

Comment:
Oh, these tests should probably use `if ! command -v dirmngr > /dev/null; then` rather than an explicit file path (same for `gpg` above :disappointed:).